tests/contracts/chain-call-balance-2/output_1.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scilla_major_version": "0",
+  "gas_remaining": "7624",
+  "_accepted": "true",
+  "message": {
+    "_tag": "c_accept",
```

Comment:
The same question here. Where is `_tag : "c_accept"` coming from? Shouldn't this be instead `c_noaccept` as in `m = { _tag : "c_noaccept"; _amount : _amount; _recipient : addrC };`